_posts/12-05-01-Building-your-Application.md

```diff
@@ -24,33 +24,33 @@ Među zadacima koje biste možda želeli da automatizujete su:
 ### Alati za automatizaciju izgradnje
 
 Alati za izgradnju mogu biti opisani kao skup skripti koje rukuju zajedničkim zadacima razvoja softvera. Alat za

```

Comment:
Ova prva rečenica je potpuna promenila smisao originala (https://github.com/codeguy/php-the-right-way/blob/gh-pages/_posts/12-05-01-Building-your-Application.md#build-automation-tools). 

"Razvoj" umesto "deploy". Dakle ja bih to ovako preveo: "Alati za izgradnju (build) mogu biti shvaćeni kao skup skripti koje se bave nekim učestalim zadacima isporuke (deploy-a) softvera"